modules/core/src/main/java/org/apache/ignite/cache/query/SpiQuery.java

```diff
@@ -24,10 +24,13 @@
 import org.apache.ignite.spi.indexing.IndexingSpi;
 
 /**
+ * This class is DEPRECATED and will be removed in the future
```

Comment:
Does such comment as 
// @deprecated is not used anymore
explains it enough or there are some recommendations for what user should use instead of SpiQuery class?

Reply:
```java
@deprecated Use {@link ScanQuery}, {@link SqlFieldQuery}, {@IndexQuery} instead.
```